a few updates for the foundation site. (#12)
pytexas/pytexas.github.io: foundation/docs/foundation/about.md

summary: Who we are
---
# About Us

> The PyTexas Foundation is a 501(c)3 non-profit run by a Texas-based volunteer 

Reviewer: Any specific reason to wrap other than aesthetics?

group. We are Python enthusiasts that want to share the programming language 
with the world, starting right here in Texas. We host and support various programs
that benefit the Python Community in Texas. We host an annual conference and 
monthly meetup, maintain a virtual community for the Python developers in Texas

> and beyond, and support various meetups in cities throughout the state.

Reviewer 2: Maybe change `various programs` to `a variety of` or eliminate the repeated `various` in `support various meetups` to `support meetups`

Author: Done

> programs that benefit the Python Community in Texas. We host an annual conference 
> and monthly meetup, maintain a virtual community for the Python developers in Texas
> and beyond, and support various meetups in cities throughout the state.

Reviewer: ```suggestion
programs that benefit the Python community in Texas, including an annual conference 
and virtual monthly meetup, a virtual community for the Python developers in Texas
and beyond, and various in-person meetups in cities throughout the state.
```

## History
Founded in 2014, meetup organizers from Houston, Austin, and San Antonio came

> together to establish a non-profit organization who's mission would be to 

Reviewer: ```suggestion
together to establish a non-profit organization whose mission would be to 
```

> serve the Python Community In Texas. Glen Zangirolami, Jeremy Boyd, and Kevin

Reviewer: ```suggestion
serve the Python community in Texas. Glen Zangirolami, Jeremy Boyd, and Kevin
```

> Horn were the Foundation's founders and first Board of Directors. In 2022 Mason

Reviewer: ```suggestion
Horn were the Foundation's founders and first Board of Directors. In 2022, Mason
```
